apps/frontend/src/app/(protect-routes)/editor/[slug]/editor-client.tsx

```
@@ -0,0 +1,633 @@
+                    className={`min-h-[400px] text-sm ${editorMode === "markdown" ? "font-mono" : ""
+                      }`}
+                  />
+                </TabsContent>
+
+                <TabsContent value="preview" className="mt-4">
+                  <div className="bg-muted/30 min-h-[400px] rounded-md border p-4">
+                    <div className="prose prose-sm max-w-none">
+                      {body ? (
+                        editorMode === "markdown" ? (
+                          <div
+                            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)